### PR TITLE
Remove web, interactive default, tox, linting, mypy

### PR DESCRIPTION
**Pull request by cidrblock**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/852

Remove the web socket support
Add pylint and mypy
Move what could be moved into pyproject.toml
Linting as needed
Remove lint script from POC

just `tox` works for a pre PR check locally

### CODE REVIEW

File: share/ansible_navigator/markdown/welcome.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ Some things you can try from here:
 
 happy automating,
 
--ansible-navigator
+-winston
```
Comment (relrod) [inline share/ansible_navigator/markdown/welcome.md:17]: :laughing:

Comment (cidrblock) [reply share/ansible_navigator/markdown/welcome.md:17]: Shhhh :)

File: ansible_navigator/actions/inventory.py [hunk 2]
```diff
@@ -133,8 +133,7 @@ def _inventory(self, value: Dict) -> None:
     def _show_columns(self) -> List:
         if self.args.inventory_columns:
             return self.args.inventory_columns.split(",")
-        else:
-            return []
+        return []
```
Comment (relrod) [inline ansible_navigator/actions/inventory.py:136]: Oh cool, another "useless else" hater :)

File: tox.ini [hunk 3]
```diff
@@ -1,36 +1,52 @@
...
-commands =
-  black -v {toxinidir}
+[testenv:clean]
+deps = coverage
+skip_install = true
+commands = coverage erase
+           rm -rf ./{[base]mypy_tmp_dir}
 
 [testenv:linters]
 install_command = pip install {opts} {packages}
```
Comment (relrod) [inline tox.ini:26]: Just curious...not changed in this PR, but why is this line necessary? Shouldn't it default to something similar?

**Review by relrod — COMMENTED** [review]
Looks fine to me. +0 on merging until smoke tests are back in CI. +1 on merging once they are back and pass.

**Comment by cidrblock** [comment]
recheck

File: share/ansible_navigator/markdown/welcome.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ Some things you can try from here:
 - `:config`                                               Explore the current Ansible configuration
 - `:collections`                                          Explore installed collections
 - `:doc <plugin>`                                         Show a plugin doc
-- `:r, :run <playbook> -i <inventory> -m interactive`     Run a playbook in interactive mode
+- `:run <playbook> -i <inventory>`                        Run a playbook in interactive mode
```
Comment (ganeshrn) [inline share/ansible_navigator/markdown/welcome.md:8]: Replace `interactive` with `stdout` 
` Run a playbook in stdout (classic) mode`

File: share/ansible_navigator/markdown/welcome.md (hunk 1 above)
Comment (ganeshrn) [inline share/ansible_navigator/markdown/welcome.md:17]: Are we renaming again :-)

Comment (cidrblock) [reply share/ansible_navigator/markdown/welcome.md:17]: nothing to see here :)

File: share/ansible_navigator/markdown/welcome.md (hunk 4 above)
Comment (ganeshrn) [reply share/ansible_navigator/markdown/welcome.md:8]: That's right, I missed that it is an interactive prompt description

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
recheck

**Pull request merged**